couchbase/content.md

```diff
@@ -0,0 +1,258 @@
+
+All of the example commands below will assume you are using volumes mapped to host directories.
+
+*SELinux workaround*
+
+If you have SELinux enabled, mounting host volumes in a container requires an extra step.  Assuming you are mounting the `~/couchbase` directory on the host filesystem, you will need to run the following command once before running your first container on that host:
+
+```
+mkdir ~/couchbase && chcon -Rt svirt_sandbox_file_t ~/couchbase
+```
+
+## Ulimits
+
+Couchbase normally expects the following changes to ulimits:
+
+```
+ulimit -n 40960        # nofile: max number of open files
+ulimit -c unlimited    # core: max core file size
+ulimit -l unlimited    # memlock: maximum locked-in-memory address space
+```
+
+These ulimit settings are necessary when running under heavy load;  but if you are just doing light testing and development, you can omit these settings and everything will still work.
+
+In order to set the ulimits in your container, you will need to run Couchbase Docker containers with the following additional `--ulimit` flags:
+
+```
+docker run -d --ulimit nofile=40960:40960 --ulimit core=100000000:100000000 --ulimit memlock=100000000:100000000 couchbase/server
+```
+
+Since `unlimited` is not supported as a value, it sets the core and memlock values to 100 GB.  If your system has more than 100 GB RAM, you will want to increase this value to match the available RAM on the system.
+
+NOTE: the `--ulimit` flags only work on Docker 1.6 or later.
+
+# Common Deployment Scenarios
+
+## Single host, single container
+
+```
+┌───────────────────────┐                                                      
+│   Host OS (Linux)     │                                                      
+│  ┌─────────────────┐  │                                                      
+│  │  Container OS   │  │                                                      
+│  │    (CentOS)     │  │                                                      
+│  │  ┌───────────┐  │  │                                                      
+│  │  │ Couchbase │  │  │                                                      
+│  │  │  Server   │  │  │                                                      
+│  │  └───────────┘  │  │                                                      
+│  └─────────────────┘  │                                                      
+└───────────────────────┘                                                      
+```
+
+This is a quick way to try out Couchbase Server on your own machine with no installation overhead - just *download and run*. In this case, any networking configuration will work; the only real requirement is that port 8091 be exposed so that you can access the Couchbase Admin Console.
+
+**Start the container**
+
+```
+docker run -d -v ~/couchbase:/opt/couchbase/var -p 8091:8091 couchbase/server
```

Comment:
`s!/server!!` --will probably need to check the rest of the doc to update to the "official image" :smile:.